mussidae/data/time-ranges/README.md

```diff
@@ -0,0 +1,12 @@
+Files
```

Comment:
Should the parent directory be named `time-ranges-processed` or `time-ranges-normalized` to distinguish it more clearly from `time-range-sources`?